packages/message-parser/tests/url.test.ts

```diff
@@ -24,8 +24,8 @@ test.each([
       ]),
     ],
   ],
-  ['http:/rocket.chat/teste', [paragraph([plain('http:/rocket.chat/teste')])]],
-  ['http:/rocket.chat/', [paragraph([plain('http:/rocket.chat/')])]],
+  ['http:/rocket.chat/teste', [paragraph([link('http:/rocket.chat/teste')])]],
+  ['http:/rocket.chat/', [paragraph([link('http:/rocket.chat/')])]],
```

Comment:
I think it is a mistake. The host name must be prefixed by `//` to be recognized as such by the `http` URL schema (e.g. `http://rocket.chat` instead of `http:/rocket.chat`).

Reply:
I got it, but I've checked and the modern browsers ignores this type of mistakes, because it's considered a common mistake, I still think that we could do the same.

Reply:
I tend to agree, but since we're not doing a full string match, there is a good chance of it falling into false positives. I believe it's acceptable as we remain open to roll it back. The most common Markdown flavors don't accept it: http:/rocket.chat